src/api/migrations/20190622162643-create-article.js

```diff
@@ -0,0 +1,51 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Articles', {
```

Comment:
@nkpremices this should already be on develop due to @MCFrank16's update article PR. Please, rebase/merge with develop.

Reply:
Yeah I will do that. I will remove this file.

Reply:
I've just noticed that he didn't allow the article category to be null. 
Do you agree with that @MemunaHaruna ?

Reply:
yes @nkpremices